locale/zh-tw/get-involved/index.md

```diff
@@ -8,6 +8,11 @@ layout: contribute.hbs
 ## 社群討論
 
 * [GitHub issues 清單](https://github.com/nodejs/node/issues)是討論 Node.js 核心功能的好地方。
+* 想實时聊聊關於 Node.js 開發方面的內容，可以使用以下之一的平臺：
+  * 使用 IRC，請使用 [IRC用戶端](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients)，轉到 `irc.libera.chat` 中的 `#node.js` 頻道，或者在您的瀏覽器中使用 [Web用戶端](https://kiwiirc.com/nextclient/) 直接連接即可。
```

Comment:
```suggestion
  * 使用 IRC，請使用 [IRC 用戶端](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients)，轉到 `irc.libera.chat` 中的 `#node.js` 頻道，或者在您的瀏覽器中使用 [Web 用戶端](https://kiwiirc.com/nextclient/) 直接連接即可。
```

Reply:
@PeterDaveHello：Yes, this will be fixed to add a space between the English and Chinese characters.